code/modules/paperwork/paper.dm

```diff
@@ -473,7 +473,10 @@
 			return
 
 		user.unEquip(src)
-		user.visible_message("<span class='danger'>[user] lights [src] ablaze with [P]!</span>", "<span class='danger'>You light [src] on fire!</span>")
+		if(istype(P, /obj/item/nullrod/godhand))
+			user.visible_message("<span class='danger'>[user] touches [src] with a finger and it bursts into flames!</span>", "<span class='danger'>You ignite [src] with your finger!</span>")
+		else
+			user.visible_message("<span class='danger'>[user] lights [src] ablaze with [P]!</span>", "<span class='danger'>You light [src] on fire!</span>")
```

Comment:
I'm not a huge fan of these snowflake checks for godhand here -- this should probably be either a signal, or a `light_with()` proc scoped to objects.